src/type/index.ts

```diff
@@ -22,3 +22,4 @@
 export { FetchOptions } from "./FetchOptions";
 export { RedirectOptions } from "./RedirectOptions";
 export { AccessBaseOptions } from "./AccessBaseOptions";
+export { AccessModes } from "./AccessModes";
```

Comment:
Aside: I think this should probably all be `export type` declarations instead maybe? I don't really know if there's that much of a difference though.